sidecar/pkg/sidecar/sidecar_utils.go

```diff
@@ -46,9 +47,23 @@ func init() {
 }
 
 // StartSideCar instantiates kube clients and starts the side-car controller
-func StartSideCar(tenantName string, secretName string) {
+func StartSideCar(tenantName string) {
 	log.Println("Starting Sidecar")
-	cfg, err := rest.InClusterConfig()
+	var cfg *rest.Config
+	var err error
+
+	if os.Getenv("DEV_NAMESPACE") != "" {
+		klog.Info("DEV_NAMESPACE present, running dev mode")
```

Comment:
in Controller the debug env variable is called `DEV_MODE`, would be nice to keep it simple with a single env variable to trace

https://github.com/minio/operator/blob/63461a1bad992d2485e4985947702d63c4c52f9c/pkg/controller/controller.go#L97


```suggestion
	var cfg *rest.Config
	var err error

	if os.Getenv("DEV_MODE") != "" {
		klog.Info("DEV_MODE present, running dev mode")
```